lib/analyzer/image-inspector.ts

```diff
@@ -155,6 +171,22 @@ async function extractImageDetails(targetImage: string): Promise<ImageDetails> {
   };
 }
 
+function isLocalImageSameArchitecture(
+  platformOption: string,
+  inspectResultArchitecture: string,
+): boolean {
+  let platformArchitecture: string;
+  try {
+    // Note: this is using the same flag/input pattern as the new Docker buildx: eg. linux/arm64/v8
+    platformArchitecture = platformOption.split("/")[1];
```

Comment:
are we sure that's the correct logic?

Reply:
I can't find an official format anywhere, but in all examples for `docker pull` or `docker buildx` the format is <os>/<arch>/<variant,version> and based on @garethr 's slack: 

> Assuming we’re happy to support multiple architectures while we’re supporting ARM64, the valid values should be:
> linux/amd64
> linux/arm64
> linux/riscv64
> linux/ppc64le
> linux/s390x
> linux/386
> linux/arm/v7
> linux/arm/v6